src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1135 @@
+      "        inequality constraints with only lower bounds:        0\n",
+      "   inequality constraints with lower and upper bounds:        0\n",
+      "        inequality constraints with only upper bounds:        0\n",
+      "\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "   0  0.0000000e+00 1.49e+06 1.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "   1  0.0000000e+00 1.35e+04 2.97e-01  -1.0 3.59e+00    -  9.90e-01 9.91e-01h  1\n",
+      "   2  0.0000000e+00 6.63e-04 9.99e+00  -1.0 3.56e+00    -  9.90e-01 1.00e+00h  1\n",
+      "   3  0.0000000e+00 1.56e-08 8.73e+01  -1.0 2.91e-04    -  9.90e-01 1.00e+00h  1\n",
+      "\n",
+      "Number of Iterations....: 3\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Constraint violation....:   7.2759576141834259e-12    1.5599653124809265e-08\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   7.2759576141834259e-12    1.5599653124809265e-08\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 4\n",
+      "Number of objective gradient evaluations             = 4\n",
+      "Number of equality constraint evaluations            = 4\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 4\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 3\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.030\n",
+      "Total CPU secs in NLP function evaluations           =      0.061\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Solve the model\n",
+    "results = solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check solver solve status\n",
+    "from pyomo.environ import TerminationCondition\n",
+    "assert results.solver.termination_condition == TerminationCondition.optimal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Analyze the results of the square problem\n",
+    "\n",
+    "\n",
+    "What is the total operating cost? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "operating cost = $ 46887895.524803445  per year\n"
```

Comment:
Use `string.format` or an f string to print the operating cost. You probably want to change units to million $/year (either using Pyomo or just multiplying by `1e-6`) and reduce the number of digits printed.